test/test-cases/regression/issue-1850.json

```diff
@@ -44,7 +44,7 @@
     "rules": [
       "SecRuleEngine On",
       "SecDefaultAction \"phase:1,status:404,deny\"",
-      "SecRule REQUEST_URI \"@contains /\" \"id:2000001,phase:1,log,redirect:'http://1.1.1.1/failed.html',t:none,msg:\"Unauthorized administrator request'\""
+      "SecRule REQUEST_URI \"@contains /\" \"id:2000001,phase:1,log,redirect:'http://1.1.1.1/failed.html',t:none,msg:'Unauthorized administrator request'\""
```

Comment:
Same here. Should have been a parsing error. Maybe open a Bug?

Reply:
Same as [here](https://github.com/owasp-modsecurity/ModSecurity/pull/3080/files/ed811f106296ab52f8b2ec3bf76b30cfd9e5ac50).